### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by AlwinEsch**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          WINSTORE: -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION="10.0.17763.0"
+
+    workspace:
+      clean: all
+
+    steps:
+
+    - script: |
+        cd ..
+        git clone --branch master --depth=1 https://github.com/xbmc/xbmc.git kodi
```
Comment (garbear) [inline azure-pipelines.yml:49]: We'll need to clone a retroplayer branch to pull in the Shader Preset API. I'll push a branch named `retroplayer-19alpha1` to https://github.com/kodi-game/xbmc.git

Comment (garbear) [reply azure-pipelines.yml:49]: Pushed: https://github.com/kodi-game/xbmc/tree/retroplayer-19alpha1

**Comment by garbear** [comment]
Also I wouldn't think this would build without the Matrix code updates in https://github.com/kodi-game/game.shader.presets/pull/2. I should have PR'd those separately

**Comment by AlwinEsch** [comment]
This request was mainly about creation of the Azure interface (auto created by Azure itself) that he know that somewhere a CI script is present.

The request can also be closed, as you have also on your request.

**Pull request merged**